### PR TITLE
feat: i added some paragraphs with different text changing tags

### PR DESCRIPTION
**Pull request by antonia2203**

### CODE REVIEW

**Review by flivanov92 — APPROVED** [review]
👍

File: src/antonia2203/text.pug [hunk 1]
```diff
@@ -0,0 +1,35 @@
+html 
+  head 
+    title Text tags 
+  body
+    p
+    i This is italic.
```
Comment (MariaTeo) [inline src/antonia2203/text.pug]: need to be indented to be a child for p tag

File: src/antonia2203/text.pug [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    p 
+    em The em tag is used to define emphasized text.
```
Comment (MariaTeo) [inline src/antonia2203/text.pug]: same

File: src/antonia2203/text.pug [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    p 
+    u The content inside it is typically displayed with an underline.
```
Comment (MariaTeo) [inline src/antonia2203/text.pug]: same

File: src/antonia2203/text.pug [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    p 
+    s The tag specifies text that is no longer correct.
```
Comment (MariaTeo) [inline src/antonia2203/text.pug]: same

File: src/antonia2203/text.pug [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    h1 
+    del This is the del element
```
Comment (MariaTeo) [inline src/antonia2203/text.pug]: same

File: src/antonia2203/text.pug [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    p 
+    | The most important rule is 
+    strong never go out
```
Comment (MariaTeo) [inline src/antonia2203/text.pug]: same

File: src/antonia2203/text.pug [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+    p 
+    s The tag specifies text that is no longer correct.
+    h1 
+    del This is the del element
+    p 
+    | The most important rule is 
+    strong never go out 
+    | after 12.
+    p
+    | This is normal text and this is
```
Comment (MariaTeo) [inline src/antonia2203/text.pug]: same

File: src/antonia2203/text.pug [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+    s The tag specifies text that is no longer correct.
+    h1 
+    del This is the del element
+    p 
+    | The most important rule is 
+    strong never go out 
+    | after 12.
+    p
+    | This is normal text and this is 
+    b bold
```
Comment (MariaTeo) [inline src/antonia2203/text.pug]: same

File: src/antonia2203/text.pug [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+    h1 
+    del This is the del element
+    p 
+    | The most important rule is 
+    strong never go out 
+    | after 12.
+    p
+    | This is normal text and this is 
+    b bold 
+    | text.
```
Comment (MariaTeo) [inline src/antonia2203/text.pug]: same

File: src/antonia2203/text.pug [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+    p 
+    | This is a regular size text -
```
Comment (MariaTeo) [inline src/antonia2203/text.pug]: 24-26 same

File: src/antonia2203/text.pug [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+    small and this is a smaller text
+    | .
+    p 
+    | The chemical formula for water is H
+    sub 2
+    | 0 .
```
Comment (MariaTeo) [inline src/antonia2203/text.pug]: same

**Review by MariaTeo — CHANGES_REQUESTED** [review]
indent

**Review by MariaTeo — APPROVED** [review]
super

**Pull request merged**